modules/UI/videolayout/RemoteVideo.js

```diff
@@ -591,17 +520,29 @@ RemoteVideo.prototype.addRemoteStreamElement = function (stream) {
 
     // Add click handler.
     let onClickHandler = (event) => {
-        let source = event.target || event.srcElement;
-
-        // ignore click if it was done in popup menu
-        if ($(source).parents('.popupmenu').length === 0) {
+        const $source = $(event.target || event.srcElement);
+        const { classList } = event.target;
+
+        const clickedOnPopover
+            = $source.parents('.connection-info').length > 0;
+        const clickedOnPopoverTrigger
+            = $source.parents('.popover-trigger').length > 0
+                || classList.contains('popover-trigger');
+        const clickedOnRemoteMenu
+            = $source.parents('.remotevideomenu').length > 0;
+
+        const ignoreClick = clickedOnPopoverTrigger
+            || clickedOnPopover
+            || clickedOnRemoteMenu;
+
+        if (!ignoreClick) {
             this.VideoLayout.handleVideoThumbClicked(this.id);
         }
 
         // On IE we need to populate this handler on video <object>
         // and it does not give event instance as an argument,
         // so we check here for methods.
-        if (event.stopPropagation && event.preventDefault) {
+        if (event.stopPropagation && event.preventDefault && !ignoreClick) {
```

Comment:
Same as LocalVideo, InlineDialog is not a child dom element of RemoteVideo so prevent InlineDialog clicks from causing focus.